### ISSUE TITLE
Cell text gets trimmed

### ISSUE THREAD

**Issue by lingefjard** [shuchkin/simplexlsxgen#124]
``Hi,
When adding text with leading or trailing spaces, the text is trimmed.

Example:
```php
$cellData = [[' text  ']];
Shuchkin\SimpleXLSXGen::fromArray($cellData)->downloadAs('workbook.xlsx');
``` 

Produces "text" in cell A1, instead of "&nbsp;text&nbsp;&nbsp;".

**Comment by shuchkin** [comment]
Use [setColWidth](https://github.com/shuchkin/simplexlsxgen#formatting) instead of aligning with spaces

**Comment by lingefjard** [comment]
Alignment is not the issue. This is about preserving data integrity.

**Comment by shuchkin** [comment]
fixed in 1.3.16

**Comment by lingefjard** [comment]
Wonderful, thanks for the swift response!
